### PR TITLE
[IMPROVE] Enable multiline for Android tablets

### PR DESCRIPTION
**Pull request by gerzonc**
## Proposed changes

Changes how keyboard behaves on Android tablets in order to enable multiline message box; now it behaves just like the phone version
## How to test or reproduce

Test keyboard usage with or without physical keyboard, just to ensure it is working properly.
## Screenshots
<table>
  <thead>
    <tr>
      <th>Before</th>
      <th>After</th>
      </thead>
    </tr>
    <tbody>
      <tr>
        <td>
        image
        </td>
        <td>
        image
        </td>
      </tr>
    </tbody>
</table >

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [ ] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by diegolmello — CHANGES_REQUESTED** [review]
> Changes how keyboard behaves on Android tablets 

What are the changes?
Can you add videos to show the difference in behavior?

File: app/containers/MessageBox/index.tsx [hunk 1]
```diff
@@ -1040,9 +1040,7 @@ class MessageBox extends Component<IMessageBoxProps, IMessageBoxState> {
 		const isAndroidTablet: Partial<IThemedTextInput> =
 			isTablet && isAndroid
 				? {
-						multiline: false,
-						onSubmitEditing: this.submit,
-						returnKeyType: 'send'
+						onSubmitEditing: this.submit
```
Comment (diegolmello) [inline app/containers/MessageBox/index.tsx:1043]: If it behaves like the smartphone version, why do we need `isAndroidTablet`?

**Review by diegolmello — CHANGES_REQUESTED** [review]

File: app/containers/MessageBox/index.tsx (hunk 1 above)
Comment (gerzonc) [reply app/containers/MessageBox/index.tsx:1043]: Lmao true

**Comment by GleidsonDaniel** [comment]
This is a change that changes the behavior of message sending.
Since now it not only "enables" the multiline, it also disables sending with enter, because the enter now jumps the text line as it is done in the mobile app.

I don't see any problems, but was that the purpose of the task from the beginning? I'm talking about changing the behavior of the app.

@diegolmello

**Comment by gerzonc** [comment]
> This is a change that changes the behavior of message sending. Since now it not only "enables" the multiline, it also disables sending with enter, because the enter now jumps the text line as it is done in the mobile app.
> 
> I don't see any problems, but was that the purpose of the task from the beginning? I'm talking about changing the behavior of the app.
> 
> @diegolmello

Yes and no. Originally we were looking for a way to make `MessageBox` to work with `multiline`, but this is not possible without changing how the keyboard behaves on Android 😭 It's been an open issue for a couple of years now on the react-native repo and sadly, this is the best solution. In any case, I think not only we should do it but as well as to evaluate the user experience so is not a complication for users later on.

**Comment by diegolmello** [comment]
> > This is a change that changes the behavior of message sending. Since now it not only "enables" the multiline, it also disables sending with enter, because the enter now jumps the text line as it is done in the mobile app.
> > I don't see any problems, but was that the purpose of the task from the beginning? I'm talking about changing the behavior of the app.
> > @diegolmello
> 
> Yes and no. Originally we were looking for a way to make `MessageBox` to work with `multiline`, but this is not possible without changing how the keyboard behaves on Android 😭 It's been an open issue for a couple of years now on the react-native repo and sadly, this is the best solution. In any case, I think not only we should do it but as well as to evaluate the user experience so is not a complication for users later on.

It's still single line because we found the issue you're talking about back then when we implemented it.
If the issue is still there, nothing changed.
I don't think we should merge this.

**Comment by gerzonc** [comment]
> > > This is a change that changes the behavior of message sending. Since now it not only "enables" the multiline, it also disables sending with enter, because the enter now jumps the text line as it is done in the mobile app.
> > > I don't see any problems, but was that the purpose of the task from the beginning? I'm talking about changing the behavior of the app.
> > > @diegolmello
> > 
> > 
> > Yes and no. Originally we were looking for a way to make `MessageBox` to work with `multiline`, but this is not possible without changing how the keyboard behaves on Android 😭 It's been an open issue for a couple of years now on the react-native repo and sadly, this is the best solution. In any case, I think not only we should do it but as well as to evaluate the user experience so is not a complication for users later on.
> 
> It's still single line because we found the issue you're talking about back then when we implemented it. If the issue is still there, nothing changed. I don't think we should merge this.

I'll close this then and we'll retry on the future.

**Pull request closed without merging**